src/components/one-time-donation/RegisterDialog.tsx

```diff
@@ -28,10 +29,12 @@ export default function RegisterForm() {
     confirmPassword: formik.values.confirmPassword as string,
     terms: formik.values.terms as boolean,
     gdpr: formik.values.gdpr as boolean,
+    picture: '',
   }
   const onClick = async () => {
     try {
       setLoading(true)
+      values.picture = assignRandomProfilePicture().trim()
```

Comment:
This random assignment will most probably throw hydration error as it will yield one avatar on the backend and another when it runs on the client.

We need to make sure that it is stored in memo or state before using it.